packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -52,41 +51,52 @@ export const ChatList = (
     IChatListItemInteractionProps &
     IChatListMenuItemsProps & {
       buttonItems?: ChatListButtonItem[];
+      chatThreadsPerPage: number;
     }
 ) => {
-  const { value } = props.dataContext as { value: GraphChat[] };
-  const chats: GraphChat[] = value;
-
   const styles = useStyles();
-  const chatClient: StatefulGraphChatListClient = useGraphChatListClient();
-  const [chatState, setChatState] = useState(chatClient.getState());
-  const [chatThreads, setChatThreads] = useState<GraphChat[]>(chats);
-  const [menuItems, setMenuItems] = useState<ChatListMenuItem[]>(props.menuItems === undefined ? [] : props.menuItems);
 
-  const onChatListEvent = (state: ChatListEvent) => {
-    // TODO: implementation will happen later, right now, we just need to make sure messages are coming thru in console logs.
+  const [chatClient, setChatClient] = useState<StatefulGraphChatListClient | undefined>();
+  const [chatState, setChatState] = useState<GraphChatListClient | undefined>();
 
-    console.log(state.type);
-    console.log(state.message);
-  };
+  // wait for provider to be ready before setting client and state
+  useEffect(() => {
+    const provider = Providers.globalProvider;
+    provider.onStateChanged(evt => {
```

Comment:
Nit: StatefulGraphChatListClient.ts does this all in one line.
```suggestion
    Providers.globalProvider.onStateChanged(evt => {
```

Reply:
The formatting is done by Prettier. If we don't follow its styling, it wouldn't let me commit.